File: source/java/com/glympse/java/hal/GSemaphore.java
//------------------------------------------------------------------------------
//
// Copyright (c) 2014 Glympse Inc.  All rights reserved.
//
//------------------------------------------------------------------------------

package com.glympse.java.hal;

import java.util.*;
import com.glympse.java.core.*;
import com.glympse.java.hal.*;
    
/*J*/public/**/ interface GSemaphore extends GCommon
{
    public void acquire();
        
    public void notify(int count);    
};  
    
/*C*typedef GSemaphore  GSemaphore;**/
    

File: source/csharp/glympse/hal/GSemaphore.cs
//------------------------------------------------------------------------------
//
// Copyright (c) 2014 Glympse Inc.  All rights reserved.
//
//------------------------------------------------------------------------------

using System;
using System.Collections;
using System.Collections.Generic;
using System.Text;
using System.Threading;

namespace Glympse 
{
    
/*J*public**/ interface GSemaphore : GCommon
{
     void acquire();
        
     void notify(int count);    
};  
    
/*C*typedef GSemaphore  GSemaphore;**/
    
}

